packages/jsii-java-runtime-test/project/src/test/java/software/amazon/jsii/testing/ComplianceTest.java

```diff
@@ -927,6 +929,18 @@ public void classWithPrivateConstructorAndAutomaticProperties() {
         assertEquals("Hello", obj.getReadOnlyString());
     }
 
+    @Test
+    public void nullShouldBeTreatedAsUndefined() {
+        NullShouldBeTreatedAsUndefined obj = new NullShouldBeTreatedAsUndefined("hello", null);
```

Comment:
I'm not even sure the `Jackson` configuration we have would include the `null`s in the generated JSON. But I reckon the intent is still validated, right?

Reply:
The POJOs get serialized to JSON before they're sent to the kernel, right? That JSON :)

Reply:
Got it. null is serialized to "null"